src/elfpy/strategies/basic.py

```diff
@@ -14,6 +14,10 @@ class BasicPolicy(User):
     most basic policy setup
     """
 
+    def __init__(self, market, rng, budget=1000, verbose=False):
```

Comment:
Can you fix the arg order so it matches the parent class

Reply:
moved verbose to the end